src/commands/createWebApp/createWebApp.ts

```diff
@@ -30,7 +30,7 @@ function isSubscription(item?: AzExtParentTreeItem): boolean {
 
 export async function createWebApp(context: IActionContext & Partial<ICreateChildImplContext>, node?: SubscriptionTreeItemBase | undefined, _nodes?: (SubscriptionTreeItemBase | undefined)[], suppressCreatedWebAppMessage: boolean = false): Promise<SiteTreeItem> {
     if (!node || !isSubscription(node)) {
-        node = <SubscriptionTreeItemBase>await ext.rgApi.tree.showTreeItemPicker(SubscriptionTreeItemBase.contextValue, context);
+        node = <SubscriptionTreeItemBase>await ext.rgApi.tree.showTreeItemPicker(new RegExp(SubscriptionTreeItemBase.contextValue, 'i'), context);
```

Comment:
Can you explain why you had to change this to a regex?

Reply:
This line was hanging during tests causing fails, my hunch is that the context value of the sub has changed over time and may have new values being appended to it.  Using a regexp here allows us to still find the match without it being a direct 1:1.  I cannot confirm for sure without digging in and adding breakpoints into the picker logic, however, I think this is very likely the case seeing that this fixed the hanging issue.

Reply:
This was a workaround for an issue I was running into during parallel tests.  I thought I had found the solution, but upon closer investigation I believe a more robust solution is needed.  Will contact you to talk more about this offline.